zstash/update.py

```diff
@@ -88,16 +83,21 @@ def update():  # noqa: C901
             cur.execute(u"select value from config where arg=?", (attr,))
             value = cur.fetchone()[0]
             setattr(config, attr, value)
-    # FIXME: Incompatible types in assignment (expression has type "bool", variable has type "None") mypy(error)
-    config.maxsize = int(config.maxsize)  # type: ignore
-    # FIXME: Incompatible types in assignment (expression has type "bool", variable has type "None") mypy(error)
-    config.keep = bool(int(config.keep))  # type: ignore
+    if config.maxsize:
```

Comment:
Same comment about if we need to check for `config.maxsize` since there is not `optional.add_argument` for `maxsize`